vm.go

```diff
@@ -518,6 +518,14 @@ func (session *ioSession) ForwardStdin(frame *api.Frame) error {
 	return vm.hyperHandler.SendIoMessage(msg)
 }
 
+// TerminateShim forces the shim to exit
+func (session *ioSession) TerminateShim() error {
+	exitCode := uint8(0)
```

Comment:
Because that's a case where the container process needs to be killed but it has not been started, therefore we can consider the "process" has returned 0.